From f27b258645cca00593715c51dab0523a5e760818 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Sat, 3 Dec 2022 12:35:16 +0100
Subject: [PATCH attr 2/7] build: Fix cross-compilation on musl-based systems

The gettext implementation in musl does not define some of the internal
symbols that AM_GNU_GETTEXT used to check for. This got fixed in gettext
0.19.8, but the AM_GNU_GETTEXT_VERSION macro requires a specific version
even if the system contains a higher one. We switch to use the new
AM_GNU_GETTEXT_REQUIRE_VERSION macro which requires at least a specific
version, and bump the AM_GNU_GETTEXT_VERSION to the first one that had
support for the new macro.

Origin: vendor
Forwarded: https://lists.nongnu.org/archive/html/acl-devel/2022-12/msg00003.html
Signed-off-by: Guillem Jover <guillem@hadrons.org>
---
 configure.ac | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 98477b5..00d2a1b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -37,7 +37,10 @@ AC_SYS_LARGEFILE
 AM_PROG_AR
 LT_INIT
 
-AM_GNU_GETTEXT_VERSION([0.18.2])
+dnl Minimal version supporting AM_GNU_GETTEXT_REQUIRE_VERSION.
+AM_GNU_GETTEXT_VERSION([0.19.6])
+dnl Require at least the following version, but use the latest available one.
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.19.8])
 AM_GNU_GETTEXT([external])
 
 AC_ARG_ENABLE([debug],
-- 
2.47.2

